mmaction/models/common/affine_layer.py

```diff
@@ -0,0 +1,25 @@
+import torch
+import torch.nn as nn
+from mmcv.cnn import NORM_LAYERS
+from torch.nn.parameter import Parameter
+
+
+@NORM_LAYERS.register_module()
+class AffineLayer(nn.Module):
```

Comment:
By the way, I don't know why we need an AffineLayer. I think merging BN into Convs is OK for finetuning on AVA. Maybe we can study this problem after you got some performance numbers on AVA.